### ISSUE TITLE
CSS filters do not work on individual SVG elements

### ISSUE THREAD

**Issue by generic-github-user** [generic-github-user/Caesium#23]
Filters work on the entire SVG element. Was going to apply variable _filter: brightness_ to node _\<circle>_ SVG/HTML elements to represent the node values. We can either use [SVG filters](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/filter) or place each node element in its own _\<div>_ element with a filter.

**Comment by generic-github-user** [comment]
The SVG elements are hidden if they are placed inside a _\<div>_ . . . and creating a new SVG element for each node is not feasible. We probably have to go with SVG filters.

**Comment by generic-github-user** [comment]
It may be easier to just use the size of the node to represent the value of the node, as a new filter would need to be created for each node element.

**Comment by generic-github-user** [comment]
This is what I am currently doing - the color of each node could also be modified with a function and applied with a style property.

**Comment by generic-github-user** [comment]
This will need to be done before the custom colors are added:
https://github.com/generic-github-user/Caesium/issues/22
Saturation could be used instead of brightness.

**Comment by generic-github-user** [comment]
Saturation should probably not be used, as nodes types with zero saturation will be almost indistinguishable from one another. It will also look uglier.

**Comment by generic-github-user** [comment]
For future expandability, it may be worth adding a filter element for each node.

**Comment by generic-github-user** [comment]
It would be easier to apply color effects if the colors were in RGB instead of hexadecimal, but it shouldn't be an issue if we use a pre-built function.

**Comment by generic-github-user** [comment]
I will probably either use the brightness value or both the the size of the nodes and the brightness for the visualization - I may add an option to the control panel where users can select one or both.

**Comment by generic-github-user** [comment]
Not technically a bug . . .

**Comment by generic-github-user** [comment]
Opened a new issue - #43
